src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/TestTypedClient.cs

```diff
@@ -15,4 +15,16 @@ public TestTypedClient(HttpClient httpClient)
 
         public HttpClient HttpClient { get; }
     }
+
+    // Simple typed client but the HttpClient parameter is missing
```

Comment:
This class will only ever be used in a single test, since this is an erroneous case. It might be better to just have it in the same file as the test itself (usually in the bottom of the file). The `TestTypedClient` above is expected to be used in multiple test files.